cmd/vicadmin/server.go

```diff
@@ -75,6 +77,82 @@ const (
 	genericErrorMessage    = "Internal Server Error; see /var/log/vic/vicadmin.log for details" // for http errors that shouldn't be displayed in the browser to the user
 )
 
+// wrapper struct around net.Conn for our custom funcs
```

Comment:
`// Conn is a wrapper struct around net.Conn that implements custom functionality`

https://blog.golang.org/godoc-documenting-go-code